lib/urlwatch/filters.py

```diff
@@ -155,6 +159,26 @@ def filter(self, data, subfilter=None):
             return data
 
 
+class BeautifierFilter(FilterBase):
+    """Beautify HTML"""
+
+    __kind__ = 'beautifier'
```

Comment:
Let's call this `beautify`, to be in line with the "verb" style of other filters (e.g. `strip` and not `stripper`, `format-json` and not `json-formatter`). Also change the class name to `BeautifyFilter` (or even `beautify-html` + `BeautifyHTMLFilter`?).

Reply:
As it beautify html, js and css, I'll stick to `Beautify`.